docs/content/python/new-roles/getting-started.md

```diff
@@ -191,121 +203,63 @@ uses [data filtering](../../guides/data_access) to translate the
+and what permissions to check.
+
+In GitClub, we initialize the session on the application
 
 {{< literalInclude
     path="examples/gitclub-sqlalchemy-flask-react/backend/app/__init__.py"
-    lines="10,14,34-40,94"
+    from="docs: begin-authorized-session"
+    to="docs: end-authorized-session"
     gitHub="https://github.com/osohq/gitclub-sqlalchemy-flask-react"
     >}}
 
-Before each request we fetch the logged in user and set the action
-depending upon the route.
-
-We can then issue regular SQLAlchemy queries to load authorized data:
+provide a helper decorator to define the permissions to check
 
 {{< literalInclude
-    path="examples/gitclub-sqlalchemy-flask-react/backend/app/routes.py"
-    from="docs: begin-repo-index"
-    to="docs: end-repo-index"
+    path="examples/gitclub-sqlalchemy-flask-react/backend/app/routes/helpers.py"
+    from="docs: begin-session-decorator"
+    to="docs: end-session-decorator"
     gitHub="https://github.com/osohq/gitclub-sqlalchemy-flask-react"
-    linenos=true
     >}}
 
-`repos` will only contain repositories that `g.current_user` is allowed to take `g.current_action` on based on our policy.
-
-### Using the basic session
-
-The basic session should be used for queries that should not have
-authorization applied. We create one using a typical SQLAlchemy `sessionmaker`.
-
-{{< literalInclude
-    path="examples/gitclub-sqlalchemy-flask-react/backend/app/__init__.py"
-    lines="4,14,29,61"
-    gitHub="https://github.com/osohq/gitclub-sqlalchemy-flask-react"
->}}
-
-Often performing actions before authorization like authentication will use
-the basic session:
-
-{{< literalInclude
-    path="examples/gitclub-sqlalchemy-flask-react/backend/app/__init__.py"
-    from="docs: begin-authn"
-    to="docs: end-authn"
-    gitHub="https://github.com/osohq/gitclub-sqlalchemy-flask-react"
-    linenos=true
->}}
-
-## Controlling access with roles
-
-Now, let's write our first rules that use role based access control. To
-setup the role library, we must:
-
-1. Initialize `OsoRoles`
-2. Persist role configuration to our database.
-3. Add role and resource configurations to our policy.
-4. Use the `Roles.role_allows` method in our policy.
-5. Assign roles to users.
-
-### Initializing `OsoRoles`
-
-`OsoRoles` extends Oso with role specific configuration & enforcement.
-We enable it by initializing the `OsoRoles` object.
+and we can then issue regular SQLAlchemy queries to load authorized data in our
+route handlers
 
 {{< literalInclude
-    path="examples/gitclub-sqlalchemy-flask-react/backend/app/__init__.py"
-    from="docs: begin-init-oso"
-    to="docs: end-init-oso"
+    path="examples/gitclub-sqlalchemy-flask-react/backend/app/routes/repos.py"
+    from="docs: begin-repo-index"
+    to="docs: end-repo-index"
     gitHub="https://github.com/osohq/gitclub-sqlalchemy-flask-react"
     linenos=true
-    hlOpts="hl_lines=7"
     >}}
 
-### Persisting roles configuration
```

Comment:
The diff is getting a bit mangled here, but basically we no longer ask people to do the persisting the config manually.

Reply:
People still need to do this though unless there is some change I missed that we made.

Reply:
Nope, my bad. Reverted.